examples/sdh-eval-workloads/ipnsw/kernel/beam_search_v2/kernel.cpp

```diff
@@ -0,0 +1,189 @@
+/*
+ * This kernel prints the Hello World message
+ */
+
+// BSG_TILE_GROUP_X_DIM and BSG_TILE_GROUP_Y_DIM must be defined
```

Comment:
Are all the versions for {Beam Search, Greedy Walk, Dot Product, Inner Product} used?

Reply:
yes